pkg/terminal/starbind/repl.go

```diff
@@ -194,7 +200,13 @@ func MakeLoad() func(thread *starlark.Thread, module string) (starlark.StringDic
 
 			// Load it.
 			thread := &starlark.Thread{Name: "exec " + module, Load: thread.Load}
-			globals, err := starlark.ExecFile(thread, module, nil, nil)
+			globals, err := starlark.ExecFileOptions(&syntax.FileOptions{
```

Comment:
Since this is the same for every invocation, can you refactor this so that there is a wrapper function around `starlark.ExecFileOptions` that encapsulates these defaults? Another option would be to create a package global which contains these defaults, but since it would be mutable we would have to ensure that it's never accidentally changed unless it's been copied.

Reply:
The wrapper function could take a pointer to a `syntax.FileOptions` as an argument and if nil, use the defaults in this function, otherwise use the one passed in. I think that's cleaner.

Reply:
Basically:

```go
func execFileOptions(opts *syntax.FileOptions, thread *Thread, filename string, src interface{}, predeclared StringDict) {
        defaults := &syntax.FileOptions{ ... }
        if opts == nil {
                opts = defaults
        }
        starlark.ExecFileOptions(opts, ...)
}
```

Reply:
Done